src/dependency-manager/src/service-config/v1.ts

```diff
@@ -57,6 +57,8 @@ export class ServiceConfigV1 extends ServiceConfig {
   name = '';
   description?: string;
   keywords?: string[];
+  image?: string;
+  ports = [8080];
```

Comment:
Per an offline convo, we should allow ports to have names so they can be referenced by upstream callers directly.